### PR TITLE
Trial PR

### PR DESCRIPTION
**Pull request by maramihali**
This is supposed to be a joke. Copyrights go to: https://www.boredpanda.com/funny-dad-jokes-puns/?utm_source=google&utm_medium=organic&utm_campaign=organic

### CODE REVIEW

File: a_file [hunk 1]
```diff
@@ -0,0 +1 @@
+If you see a robbery at an Apple Store does that make you an iWitness?
```
Comment (kele) [inline a_file:1]: I'm not gonna lie, this took me a while, but I chuckled. :D

Comment (kele) [reply a_file:1]: Please add a comment explaining why the joke is funny. It isn't obviously clear.

**Review by kele — COMMENTED** [review]
Needs more work.

File: a_file (hunk 1 above)
Comment (maramihali) [reply a_file:1]: PTAL

Comment (kele) [reply a_file:1]: Nothing has changed from what I can see.

Comment (empijei) [reply a_file:1]: Yeah, me either.

**Review by empijei — CHANGES_REQUESTED** [review]

**Pull request closed without merging**